package.json

```diff
@@ -19,6 +19,12 @@
   "bugs": {
     "url": "https://github.com/katowulf/mockfirebase/issues"
   },
+  "files": [
+    "*.md",
+    "src/",
+    "tutorials/",
+    "browser/"
```

Comment:
That makes sense. They were not explicitly ignored, so we have been deploying them. That is partially why I've grown to like using `packages.json/files` over `.npmignore`. If you forget to whitelist something, stuff breaks and you find out pretty quick, but it is pretty easy to leak a bunch of files into your distribution with a simple typo in `.npmignore` (and have it go unnoticed for a long time).